tests/epics/areadetector/test_controllers.py

```diff
@@ -35,7 +35,7 @@ async def ad(RE) -> ADSimController:
 
 async def test_ad_controller(RE, ad: ADSimController):
     with patch("ophyd_async.core.signal.wait_for_value", return_value=None):
-        await ad.arm()
+        await ad.arm(num=0)
```

Comment:
I think we should make it clear that `num=0` should be reserved for the fly scanning case. Maybe we should set `num=1` here and test it actually tries to take 1 image? So there's an assert statement later that checks it's set to 0, which should just change to being set to 1

Reply:
Done